### PR TITLE
Update Github Issue Template Workflow

### PR DESCRIPTION
**Pull request by rafayet-monon**
Closes https://github.com/nimblehq/git-template/issues/8

## What happened

Update the Github issue template workflow to a newer version.

## Insight

Currently, this repo has the older version of the issue template workflow. As Github introduced the new issue template workflow we can update to a newer version.
 
## Proof Of Work

A [test Github repository](https://github.com/rafayet-monon/git-template-issue-test) created from the [forked repository](https://github.com/rafayet-monon/git-template) and this PR branch was used as default.

[Issue templates ](https://github.com/rafayet-monon/git-template-issue-test/issues/new/choose) to choose from -

image

### CODE REVIEW

**Review by hoangmirs — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/rfc_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+name: "Request For Comments (RFC)"
```
Comment (olivierobert) [inline .github/ISSUE_TEMPLATE/rfc_template.md:2]: This template is not needed here. It was made solely to discuss process changes on Compass.

File: .github/ISSUE_TEMPLATE/content_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: "Content"
```
Comment (olivierobert) [inline .github/ISSUE_TEMPLATE/content_template.md:2]: This template is not needed here. It was made solely to propose new content pages on Compass.

**Review by olivierobert — CHANGES_REQUESTED** [review]
I think we should keep just the bug and feature templates

File: .github/ISSUE_TEMPLATE/rfc_template.md (hunk 1 above)
Comment (rafayet-monon) [reply .github/ISSUE_TEMPLATE/rfc_template.md]: deleted in [ce1d276](https://github.com/nimblehq/git-template/pull/9/commits/ce1d2767b8b1ec393a35c50a0b17c09c8cb7eb37)

File: .github/ISSUE_TEMPLATE/content_template.md (hunk 2 above)
Comment (rafayet-monon) [reply .github/ISSUE_TEMPLATE/content_template.md]: deleted in [ce1d276](https://github.com/nimblehq/git-template/pull/9/commits/ce1d2767b8b1ec393a35c50a0b17c09c8cb7eb37)

**Review by olivierobert — APPROVED** [review]

**Pull request merged**